pkg/manifests/config.go

```diff
@@ -188,7 +189,9 @@ func (c *Config) applyDefaults() {
 		c.HTTPConfig = &HTTPConfig{}
 	}
 	if c.TelemeterClientConfig == nil {
-		c.TelemeterClientConfig = &TelemeterClientConfig{}
+		c.TelemeterClientConfig = &TelemeterClientConfig{
+			TelemeterServerURL: "https://infogw.api.openshift.com/metrics/v1/receive",
```

Comment:
Can we not pass this URL via the config options?

Reply:
this is just the default, for when no option is specified